languages/ruby/exercises/concept/exceptions/simple_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '-', '/', '*']
+
+  # TODO: Show how to define custom exceptions
+  # TODO: Explain why custom errors are subclasses of StandardError
+  # TODO: Explain why is not a good idea to rescue from Exception
```

Comment:
The premise of "not a good idea to rescue from an Exception" may not be a good premise.  Instead, we would want to explain when and why to rescue from an Exception, and when and why we might re-raise from inside a rescue.

Reply:
Gotcha, I'll try to elaborate on this